_pytest/config.py

```diff
@@ -399,13 +399,15 @@ def consider_conftest(self, conftestmodule):
             self.consider_module(conftestmodule)
 
     def consider_env(self):
-        self._import_plugin_specs(os.environ.get("PYTEST_PLUGINS"))
+        specs = os.environ.get("PYTEST_PLUGINS")
+        if specs:
+            plugins = specs.split(',')
```

Comment:
If you prefer we can easily change `consider_env` to:

```python
    def consider_env(self):
        specs = os.environ.get("PYTEST_PLUGINS")
        if specs:
            specs = specs.split(',')
        self._import_plugin_specs(specs)
```

But I'm curious if you have an example where not doing so would be a problem.